spec/syntax.md

```diff
@@ -502,7 +498,7 @@ They are allowed in translatable text as well as in literals.
 
 ```abnf
 text-escape = "\\" / "\{" / "\}"
-literal-escape = "\\" / "\(" / "\)"
```

Comment:
These can probably benefit from the same refactor using a named backslash and grouping the backslash outside the set of escaped characters that @gibson042 [suggested](https://github.com/unicode-org/message-format-wg/pull/347#discussion_r1120574252) in the main ABNF PR.